README.md

```diff
@@ -380,20 +380,24 @@ changeset.rollback(); // returns changeset
 
 #### `merge`
 
-Merges 2 valid changesets and returns a new changeset with the same underlying content and validator as the origin. Both changesets must point to the same underlying object. For example:
+Merges 2 changesets and returns a new changeset with the same underlying content and validator as the origin. Both changesets must point to the same underlying object. For example:
 
 ```js
 let changesetA = new Changeset(user, validatorFn);
 let changesetB = new Changeset(user, validatorFn);
 changesetA.set('firstName', 'Jim');
+changesetA.get('errors') // [{key: 'firstName', value: 'Jim', validations: ['Firstname cannot be Jim, sorry Jim']}]
 changesetB.set('firstName', 'Jimmy');
+changesetB.get('errors') // []
 changesetB.set('lastName', 'Fallon');
 let changesetC = changesetA.merge(changesetB);
 changesetC.execute();
 user.get('firstName'); // "Jimmy"
 user.get('lastName'); // "Fallon"
 ```
 
+When merging, the changeset being merged in takes precedence. For example: if `B` contains errors, the errors from `B` will be merged, and the invalid attributes on `B` will not be merged from `A` even if they're valid on `A`. Likewise, if `A` contains errors on attributes which are valid on `B`, the value from `B` will be used and the errors for those keys will not be merged from `A`. This is to prevent the new changeset from containing `changes` and `errors` on the same keys, which is a guarantee of a normal changeset.
```

Comment:
This is still confusing, in fact I think it would be better to just remove all the changes to the readme and only keep the change on L383. I feel like this is just implementation detail that the end user doesn't need to know about.